packages/shared/src/contexts/ActiveFeedNameContext.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react';
+import { AllFeedPages } from '../lib/query';
+
+export type ActiveFeedNameContextValue = {
```

Comment:
Perhaps we can consider a `ActiveFeedNameContextProvider` so we can use it in one go (there are some examples)

And maybe expose something like this as a hook for reading value:

```
export const useActiveFeedContext = (): ActiveFeedContextValue =>
  useContext(ActiveFeedContext);
```